syz-fuzzer/fuzzer.go

```diff
@@ -290,7 +291,8 @@ func main() {
 		fetchRawCover:            *flagRawCover,
 		noMutate:                 r.NoMutateCalls,
 		stats:                    make([]uint64, StatCount),
-		newInputSem:              semaphore.NewWeighted(int64(2 * *flagProcs)),
+		// Queue no more than ~3 new inputs / proc.
+		newInputSem: semaphore.NewWeighted(int64(3 * *flagProcs)),
```

Comment:
Since we don't use as "weighted", it's easier to in Go with a channel.
You just send/receive instead of Acquire/Release.

Reply:
I've replaced it with a go channel.